db/snapshot/backup_task.cc

```diff
@@ -174,13 +174,20 @@ future<> backup_task_impl::backup_sstable(sstables::generation_type gen, const c
     }
 };
 
+backup_task_impl::sstables_manager_for_table::sstables_manager_for_table(const replica::database& db, table_id t)
+    : manager(db.get_sstables_manager(*db.find_schema(t)))
```

Comment:
I didn't dive into `find_schema()` - but from here it seems that if `find_schema()` with the given `table_id t` returns `null` - we will not satisfy `get_sstables_manager(const schema& s`) as `s` will be a reference to `null`.